1-js/02-first-steps/01-hello-world/article.md

```diff
@@ -1,87 +1,87 @@
-# Hello, world!
+# வணக்கம், உலகமே!
 
-This part of the tutorial is about core JavaScript, the language itself.
+டுடோரியலின் இந்த பகுதி ஜாவாஸ்கிரிப்ட் என்கிற நிரலாக்க மொழியின் மைய்யக் கருத்துகளை பற்றியது.
```

Comment:
Incorrect meanings

Reply:
Hey @KMuthusamyms. Thanks for the review. The translation seems fine to me. Can you suggest an edit in case of better translation?

Reply:
I have added for the following words:

1. டுடோரியலின் - Its enough
2. மைய்ய -> மையக்

Reply:
Thanks :)